src/modules/file/file.controller.ts

```diff
@@ -54,12 +57,26 @@ export class FileController {
   ) {
     const file = await this.fileService.findByIdAndUserId(id, user.id);
 
+    const filesize = await this.fileService.getFilesize(data.body);
+
+    const bytesSize = await this.redisService.get(id);
```

Comment:
You should do increment right away instead of getting size, than processing, and after it incrementing. INCBY returns incremented value. If it's over the limit - don't even bother processing any further.